src/modules/checker.js

```diff
@@ -0,0 +1,19 @@
+import taskArray from './taskArray.js';
+import saveData from './saveData.js';
+
+const check = (index) => {
+  if (taskArray[index].completed === false) {
+    taskArray[index].completed = true;
+  } else if (taskArray[index].completed === true) {
+    taskArray[index].completed = false;
+  }
+  saveData();
+};
+
+const clearCompleted = () => {
+  const arrayTask = taskArray.filter((task) => task.completed === false);
+  localStorage.setItem('tasks', JSON.stringify(arrayTask));
+  window.location.reload();
```

Comment:
[OPTIONAL] Well done overall 💯 but instead of reloading the screen which provides a bad user experience you could consider emptying the list and repopulating it 🤔 by calling the function to add the list of tasks from the `localStorage` but with the new updated `arrayTask`. Thoughts on that improvement?

Reply:
Hi @didierganthier !
Yes, of course, you are right. I had forgotten to do that refactoring, probably because I was concentrating on making the changes happen in the browser and then I forgot heh. Thank you very much, I will implement it.